packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -140,11 +140,13 @@ class SectionHeader extends React.Component {
     /* eslint-disable jsx-a11y/no-static-element-interactions */
     return (
       <Element {...attributes} onClick={onClick} className={sectionHeaderClassNames} tabIndex="0">
-        <Arrange
-          fitStart={onClick && accordionIcon}
-          fill={<span aria-hidden={(onClick !== undefined)} className={cx('title')}>{headerText}</span>}
-          className={cx('title-arrange')}
-        />
+        <div role="button" aria-expanded={isOpen} tabIndex="-1" aria-label={headerText} className={cx('arrange-wrapper')}>
```

Comment:
Do we require `tabIndex="-1"` for this div

Reply:
Yes, without which, jaws doesn't announce state of button